extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -119,6 +119,12 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return null;
   }
 
+  /** @override */
+  getSigningServiceNames_() {
+    // TODO(levitzky) Add dev key name once it goes live.
+    return getMode().localDev ? ['google'] : ['google'];
```

Comment:
[This part](https://github.com/google/amphtml/blob/690004b3339421bdcfdca6548e25423ea43a7de5/extensions/amp-a4a/0.1/amp-a4a.js#L524-L526). Instead of returning names of urls that are defined in `a4a-config.js`, just return an XHR request to the url.